chrome/content/modules/wzQuicktextVar.jsm

```diff
@@ -281,9 +287,14 @@ wzQuicktextVar.prototype = {
     return this.process_file(aVariables);
   }
 ,
-  get_image: function(aVariables)
+  get_image: function(aVariables, aType)
   {
```

Comment:
Handle the image code depending on aType (html/text) here as it is not done anymore above.